docs/docsite/Makefile

```diff
@@ -37,6 +37,7 @@ generate_rst: staticmin
 htmldocs: generate_rst
 	cd ../../lib/ansiblelint; python ./generate_docs.py
 	CPUS=$(CPUS) $(MAKE) -f Makefile.sphinx html
+	@sh -c "git diff --quiet || { echo 'ERROR: Building docs changed tracked files, include them in commit to avoid this failure.'; exit 1; }"
```

Comment:
Honestly, I'd rather get rid of the makefile and run things directly via tox.

Reply:
That worth considering but not part of this chance. Lets try to improve it one thing at a time.